chainable_api.go

```diff
@@ -366,6 +366,39 @@ func (db *DB) Scopes(funcs ...func(*DB) *DB) (tx *DB) {
 	return tx
 }
 
+func (db *DB) executeScopes(keepScopes bool) (tx *DB) {
+	tx = db.getInstance()
+	scopes := db.Statement.scopes
+	if len(scopes) == 0 {
```

Comment:
in a scope condition, it might have other scope conditions, so we are writing the code like this:

```go
for len(db.Statement.scopes) > 0 {
	db = db.executeScopes(false)
		scopes := db.Statement.scopes
		db.Statement.scopes = nil
		for _, scope := range scopes {
			db = scope(db)
		}
	}
```

Seems like the change will break it.

Reply:
Sorry, it's my fault.